packages/core/src/Provider/Provider.ts

```diff
@@ -1,5 +1,17 @@
+import { Percent } from '../Cardano';
+import { Tip } from '@cardano-ogmios/schema';
+
 export type HealthCheckResponse = {
   ok: boolean;
+  localNode?: {
+    ledgerTip?: Tip;
+    networkSync?: Percent;
+  };
+};
+
+export type ServicesHealthCheckResponse = {
```

Comment:
I think a more appropriate place for this is in `cardano-services` package as it's not per-provider, but per the http server.

Reply:
be86382cbc1aadd303a9902ff3be8ba07273ccd6